res/css/views/rooms/_UserIdentityWarning.pcss

```diff
@@ -0,0 +1,26 @@
+/*
+Copyright 2024 New Vector Ltd.
+
+SPDX-License-Identifier: AGPL-3.0-only OR GPL-3.0-only
+Please see LICENSE files in the repository root for full details.
+*/
+
+.mx_UserIdentityWarning {
+    border-top: 1px solid $separator;
+    padding-top: 5px;
+    margin-left: calc(-42px + var(--RoomView_MessageList-padding));
+    display: flex;
+    align-items: center;
+
+    .mx_BaseAvatar {
+        margin-left: 8px;
+    }
+    .mx_UserIdentityWarning_main {
+        margin-left: 24px;
+        flex-grow: 1;
+    }
+}
+
+.mx_MessageComposer.mx_MessageComposer--compact > .mx_UserIdentityWarning {
+    margin-left: calc(-25px + var(--RoomView_MessageList-padding));
```

Comment:
Where `-42px` is coming from ?

Reply:
The 42px and 25px come from `views/rooms/_MessageComposer.pcss`.  It's the `padding-left` of `.mx_MessageComposer_wrapper`, so that I can get the avatar lined up with the avatars in the timeline.

Reply:
Ok, a comment before this line is welcomed

Reply:
Added a comment before the line with -42px.  I didn't add a comment before this line, because I think that the other comment should make this obvious.  But I can add another comment if you still think it's needed.